Documentation/Bookmarks/README.md

```diff
@@ -0,0 +1,71 @@
+# Bookmarks
+
+The Bookmarks component will show the list of bookmarks in a RecyclerView and allows the user to select
```

Comment:
"shows" rather than "will show", for consistency with "allows" and so it won't need changed later

Reply:
Also, maybe say the following?

> ...the list of ***a map's*** bookmarks...